pkg/utils/kubeclient/node.go

```diff
@@ -38,7 +38,8 @@ func GetNode(client client.Client, name string) (node *v1.Node, err error) {
 	return node, err
 }
 
-// Check if the node is ready
+// IsReady checks if the node is ready
+// If the node is ready,it returns bool type 1,else,it returns bool type 0.
```

Comment:
If the node is ready, it returns true. Otherwise, it returns false.

Reply:
I have added new commits ,could you receive it?